### PR TITLE
0.2 upgrade

### PR DESCRIPTION
**Pull request by joshmoore**
Update `omero-cli-duplicate` to match https://github.com/ome/omero-cli-render/pull/1, using https://github.com/openmicroscopy/omero-test-infra to run all integration tests in Travis (now passing).

Propose releasing as 0.2.0.

cc: @dominikl @jburel @pwalczysko

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,4 +36,4 @@ licensed under the terms of the GNU General Public License (GPL) v2 or later.
 Copyright
 ---------
 
-2017, The Open Microscopy Environment
\ No newline at end of file
+2017, The Open Microscopy Environment
```
Comment (jburel) [inline README.rst]: 2018

**Comment by joshmoore** [comment]
Was there anything anyone wanted to add to `o-c-d`?

**Comment by jburel** [comment]
Note: we will need a new version when 5.4.2 is out to remove ``cli.py``

**Comment by joshmoore** [comment]
Already carded: https://trello.com/c/65fh1AUq/26-remove-pytest-command-and-clipy, but I don't think we will have to release just because 5.4.2 is out, but certainly the next time we touch these we should delete.

**Comment by jburel** [comment]
We don't need to release, just need to remember to remove it

**Comment by joshmoore** [comment]
Merging, tagging, releasing. If there are any issues, we'll go directly to `0.3`

**Pull request merged**